provider/digitalocean/digital_ocean_instance.go

```diff
@@ -51,14 +51,25 @@ func (do *DigitalOcean) CreateInstance(ctx *lepton.Context) error {
 		})
 	}
 
+	tags := make([]string, 0, len(config.CloudConfig.Tags)+2)
```

Comment:
what's this +2 for?

you can just do a ```tags:= []string{}```

Reply:
+2 is for the two default tags added (see line 55). Your option definitely looks nicer, I can amend my pull request if you want. I just wanted to exactly allocate the required memory!